frontend/tests/unit/components/inputs/NumericInput.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { mount } from '@vue/test-utils';
+
+import BasicInput from '@/components/inputs/BasicInput.vue';
+import NumericInput from '@/components/inputs/NumericInput.vue';
+import { ALPHABET_CHARACTERS, DECIMAL_CHARACTERS } from '~/utils/data_generators';
+
+function createWrapper(options?: {
+  modelValue?: string;
+  focusOnMount?: boolean;
+  attachToBody?: boolean;
+  valid?: boolean;
+  alignRight?: boolean;
+}) {
+  return mount(NumericInput, {
+    shallow: false,
```

Comment:
I would always prefer a shallow mount for unit tests. This way it can be ensured that we only test the one component that should be tested and we avoid possible side effects by a child component.

Reply:
✅